packages/admin/admin/src/table/filterbar/filterBarMoreFilters/FilterBarMoreFilters.tsx

```diff
@@ -1,47 +1,74 @@
 import { Filter } from "@comet/admin-icons";
-import { ComponentsOverrides, Theme } from "@mui/material";
-import { WithStyles, withStyles } from "@mui/styles";
+import { ComponentsOverrides } from "@mui/material";
+import { css, styled, Theme, useThemeProps } from "@mui/material/styles";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
 import * as React from "react";
 import { FormattedMessage } from "react-intl";
 
 import { FilterBarButton } from "../filterBarButton/FilterBarButton";
-import { FilterBarMoveFilersClassKey, styles } from "./FilterBarMoreFilters.styles";
 
 /**
  * @deprecated Use MUI X Data Grid in combination with `useDataGridRemote` instead.
  */
-export interface FilterBarMoreFiltersProps {
+
+export type FilterBarMoreFiltersClassKey = "root" | "button";
+
+const Root = styled("div", {
+    name: "CometAdminFilterBarMoreFilters",
+    slot: "root",
+    overridesResolver(_, styles) {
+        return [styles.root];
+    },
+})(
+    css`
+        margin-bottom: 10px;
+        margin-right: 6px;
+    `,
+);
+
+const StyledFilterBarButton = styled(FilterBarButton, {
+    name: "CometAdminFilterBarMoreFilters",
+    slot: "button",
+    overridesResolver(_, styles) {
+        return [styles.button];
+    },
+})(
+    ({ theme }) => css`
+        font-weight: ${theme.typography.fontWeightBold};
+    `,
+);
+
+export interface FilterBarMoreFiltersProps extends ThemedComponentBaseProps<{ root: any; button: typeof FilterBarButton }> {
```

Comment:
Please set the actual type

```suggestion
export interface FilterBarMoreFiltersProps extends ThemedComponentBaseProps<{
    root: "div";
    button: typeof FilterBarButton;
}> {
```